libs/util/Collections.arucas

```diff
@@ -812,7 +812,7 @@ class AsymmetricBiMap : BiMap {
             this.front.put(key, value);
         }
         if (this.containsValue(value)) {
-            this.getKey(value).add(key);
+            this.getKeys(value).add(key);
```

Comment:
This should be `getKey(value)` gets the key for a value (just a method that wraps `this.back.get(value)`.
`getKeys` is incorrect